src/current/cockroachcloud/configure-cloud-org-sso.md

```diff
@@ -162,48 +162,79 @@ You can add a custom authentication method to connect to any IdP that supports [
 
 ### OIDC
 
-To configure a custom OIDC authentication method:
+To configure a custom OIDC authentication method, you need the following information from your IdP:
 
-1. Log in to your IdP and gather the following information, which you will use to configure CockroachDB {{ site.data.products.cloud }} SSO:<ul><li>Issuer URL</li><li>Client ID</li><li>Client secret</li><li>Callback URL</li></ul>
+- Issuer URL
+- Client ID
+- Client secret
+
+These instructions work for Okta. If you use a different IdP, refer to its documentation for configuring OIDC.
+
+1. Log in to Okta as a user with the Admin role.
+1. In the Admin Console, click **Applications**, then click **Browse Catalog**.
+1. Search for "Cockroach Labs", Click **Add integration**.
+1. Set **Application label** to a name for the integration. Set Entity ID and ACS URL to `none`. These fields are ignored.
+1. Click **Next**.
+1. In Sign-On Options, select OpenID Connect. SAML is selected by default.
+1. Set **Application username format** to **Email**.
+1. Click **Done**. The app integration's details appear.
+1. Assign at least one Okta identity to the application, such as the identity you already use to sign in to CockroachDB {{ site.data.products.cloud }}. Click **Assignments**, then click **Assign to People**. Find the identity, then click **Assign**, then click Save and go back. Click Done to close the assignment dialog.
```

Comment:
```suggestion
1. Assign at least one Okta identity to the application, such as the identity you already use to sign in to CockroachDB {{ site.data.products.cloud }}. Click **Assignments**, then click **Assign to People**. Find the identity, then click **Assign**, then click **Save and go back**. Click **Done** to close the assignment dialog.
```

Not sure if you were being selective in which UI labels to bold. Suggesting just in case.